app/main.py

```diff
@@ -1,6 +1,47 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import os
+import requests
+from requests.exceptions import (
+    ConnectionError,
+    JSONDecodeError,
+    RequestException
+)
+
+
+URL = "http://api.weatherapi.com/v1/current.json?"
+API_KEY = os.environ["API_KEY"]
```

Comment:
Ensure that the `API_KEY` environment variable is set before running the script, as it is required for accessing the weather API.